content/ui-patterns/loading.mdx

```diff
@@ -369,25 +369,27 @@ However, the button that initiates the a loading process may not be semantically
 {/*<!-- _TODO: Consider moving most of the button loading state details to the [Button interface guidelines](/components/button)_ -->*/}
 </>
 
-<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{alignItems: 'start', gap: 3}}>
+<img
```

Comment:
I _think_ we determined we want to just leave the button label text the same as to not change the width of the button in the loading state. But maybe we want to offer it as an option? Either way I'm fine merging this PR and following up later after a11y review

Reply:
We can update with more details after we talk about it in a11y office hours